settings/base.py

```diff
@@ -4,7 +4,14 @@
 
 # Django settings file for a project based on the playdoh template.
 
-from funfactory.settings_base import *
+from funfactory.settings_base import (
+    path,
+    BASE_PASSWORD_HASHERS,
+    ROOT,
+    ROOT_URLCONF,
+    TEST_RUNNER
+)
```

Comment:
However, I would argue that it might be worth just copying those values. 
If you do, what the heck do you need funfactory for? You certainly won't need its crazy hacks in manage.py which shouldn't be applicable any more now that you're on the peep bandwagon.